### PR TITLE
Rename retry_exceptions to retryable_exceptions

### PR DESCRIPTION
**Pull request by greysteil**
I think this is clearer - it's an array, not a command.

### CODE REVIEW

**Comment by greysteil** [comment]
@isaacseymour - thoughts? Removes confusion from using `retry` as a noun.

File: spec/retry/constant_options_validator_spec.rb [hunk 1]
```diff
@@ -31,12 +31,12 @@
     end
 
     context 'retry exceptions' do
```
Comment (isaacseymour) [inline spec/retry/constant_options_validator_spec.rb]: And the context too!

File: spec/retry/constant_options_validator_spec.rb [hunk 2]
```diff
@@ -69,12 +69,12 @@
     end
 
     context 'bad retry exceptions' do
```
Comment (isaacseymour) [inline spec/retry/constant_options_validator_spec.rb]: Same here

File: spec/retry/variable_options_validator_spec.rb [hunk 3]
```diff
@@ -28,12 +28,12 @@
     end
 
     context 'retry exceptions' do
```
Comment (isaacseymour) [inline spec/retry/variable_options_validator_spec.rb]: and again

File: spec/retry/variable_options_validator_spec.rb [hunk 4]
```diff
@@ -71,12 +71,12 @@
     end
 
     context 'bad retry exceptions' do
```
Comment (isaacseymour) [inline spec/retry/variable_options_validator_spec.rb]: and this one

**Comment by isaacseymour** [comment]
I like it, `retry_exceptions` sounds like a function name rather than the key in the options.

**Comment by greysteil** [comment]
@isaacseymour - will leave to you to merge and bump the version (would be a confusing readme if we don't bump immediately on this one).

**Pull request merged**